modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +55,343 @@ docker compose version
+
+=== Schema Registry
+
+On the *Schema Registry* page, you see an overview of your schemas. You can create, manage, and inspect your schemas without leaving Redpanda Console.
+
+// (step {"action":"find", "selector": "[data-testid='nav-link-Schema Registry']", "matchText": "Schema Registry", "click": true})
+// (step {"action":"goTo", "url": "http://localhost:8080/schema-registry/subjects/transactions?version=latest"})
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "schema-reg.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+
+image::console:schema-reg.png[]
+
+See also: xref:console:ui/schema-reg.adoc[].
+
+=== Configure access control for Redpanda
+
+Managing access control lists (ACLs) can be complex, but Redpanda Console simplifies this with an intuitive interface.
 
-. xref:get-started:attachment$three-brokers/docker-compose.yml[Download] the following `docker-compose.yml` file on your local file system.
+This quickstart deployment includes a bootstrapped superuser with full administrative privileges. The superuser has unrestricted access to all topics, consumer groups, transactions, and cluster-level operations. By default, the credentials for the superuser are:
+
+- Username: `superuser`
+- Password: `secretpassword`
+
+For improved security and role-based access control, you should create additional users with restricted permissions as needed.
+
+On the *Security* page, you can:
+
+- Create, view, and edit ACLs for topics, consumer groups, transactions, and the entire cluster.
+- Quickly understand what each principal (user) is authorized to do.
+
+Suppose you're onboarding a new team member who needs access to specific topics and consumer groups. You can use Redpanda Console to ensure users have the right permissions to perform their tasks.
+
+. In the left menu, click *Security*.
+// (step {"action":"find", "selector": "[data-testid='nav-link-Security']", "matchText": "Security", "click": true})
+. On the *Users* tab, click *Create user*.
+// (step {"action":"find", "selector": "[data-testid='create-user-button']", "matchText": "Create user", "click": true})
+. Enter "Sasha" in the username field.
+// (step {"action":"find", "selector": "[data-testid='create-user-name']", "typeKeys": "Sasha", "click": true})
+. Enter "sashapassword" in the password field.
+// (step {"action":"find", "selector": "[data-testid='password-input-toggle']", "click": true})
+// (step {"action":"find", "selector": "[data-testid='test_field']", "setVariables": [{"name": "password","regex": ".*"}], "click": true})
+// (step {"action":"find", "selector": "[data-testid='password-input-toggle']", "click": true})
+// (step {"action":"wait"})
+// (step {"action":"saveScreenshot", "path": "user.png", "directory": "../../modules/console/images", "overwrite": "byVariation"})
+. Click *Create*.
```

Comment:
You also need to click *Done*, but then you'd get a different screenshot. I'll suggest it in next line.